apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableServiceApiClientPartial.cs

```diff
@@ -120,38 +120,4 @@ public partial class SampleRowKeysStream
             }
         }
     }
-    
-    public partial class BigtableServiceApiClientImpl
-    {
-        private const string ResourcePrefixHeader = "google-cloud-resource-prefix";
-
-        partial void Modify_ReadRowsRequest(ref ReadRowsRequest request, ref CallSettings settings) =>
-            ApplyResourcePrefixHeader(ref settings, request.TableName);
-
-        partial void Modify_SampleRowKeysRequest(ref SampleRowKeysRequest request, ref CallSettings settings) =>
-            ApplyResourcePrefixHeader(ref settings, request.TableName);
-
-        partial void Modify_MutateRowRequest(ref MutateRowRequest request, ref CallSettings settings)
-        {
-            GaxPreconditions.CheckState(request.IsIdempotent(), "Non-idempotent MutateRow requests are not allowed. Specify a version with all SetCell mutations.");
-            ApplyResourcePrefixHeader(ref settings, request.TableName);
-        }
-
-        partial void Modify_MutateRowsRequest(ref MutateRowsRequest request, ref CallSettings settings)
-        {
-            GaxPreconditions.CheckState(request.IsIdempotent(), "Non-idempotent MutateRows requests are not allowed. Specify a version with all SetCell mutations.");
```

Comment:
This needs to be kept in

Reply:
Humbug - sorry about that. I checked that they were all using ApplyResourcePrefixHeader, but I didn't check that was the *only* thing...

Reply:
(hopefully this causes a test failure)

Reply:
Nope, looks like that build passed. I'll have to add tests for this.

Reply:
Ok, so I did add the tests, they were just integration tests, so that's why we didn't see the failure: https://github.com/GoogleCloudPlatform/google-cloud-dotnet/blob/80a56f78d90be847cd5e286275889da43c0a6477/apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.IntegrationTests/MutateRowsTest.cs#L253

https://github.com/GoogleCloudPlatform/google-cloud-dotnet/blob/74c3bd7b91a788c8a456fce9fea454d1650b6ccf/apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.IntegrationTests/MutateRowTest.cs#L294